lib/models/addon.js

```diff
@@ -851,12 +851,7 @@ var Addon = CoreObject.extend({
   addonJsFiles: function(tree) {
     this._requireBuildPackages();
 
-    var includePatterns = this.registry.extensionsForType('js').map(function(extension) {
-      return new RegExp(extension + '$');
-    });
-
     return new Funnel(tree, {
-      include: includePatterns,
```

Comment:
The output tree here just dumped all of the assets in the tree into the `modules/foo-addon/` folder. That seems incredibly odd and isn't just a pass-through at that point; we've added new assets into a location that is [treated as JS-only](https://github.com/ember-engines/ember-engines/blob/master/lib/engine-addon.js#L298-L349).

I'm pretty concerned that this will break things but we can let it ride and see.